app/Http/Controllers/Api/V1/CurrikiGo/OutcomeController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers\API\V1\CurrikiGo;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\V1\CurrikiGo\GetStudentResultRequest;
+use App\Http\Resources\V1\CurrikiGo\StudentResultResource;
+use Illuminate\Http\Request;
+use App\Services\LearnerRecordStoreService;
+
+/**
+ * @group 15. CurrikiGo Outcome
+ *
+ * APIs for generating outcomes against students' submissions.
+ */
+class OutcomeController extends Controller
+{
+    /**
+     * Get Student Results Summary
+     *
+     * Fetch LRS statements based on parameters, and generate a student result summary
+     *
+     * @param GetStudentResultRequest $studentResultRequest
+     *
+     * @responseFile responses/outcome/student-result-summary.json
```

Comment:
Need to mention response code if it is not 200.

Reply:
I have made a correction to the response status, and made it 200, as I'm not creating anything there.